Business Rules/Copy Assign group and to Request/SC_TASK details to Request.js

```diff
@@ -0,0 +1,22 @@
+(function executeRule(current, previous /*null when async*/) {
+    var req = current.request.getRefRecord();
+
+    // Check if the REQ record exists
+    if (req) {
```

Comment:
This will resolve to true since `.getRefRecord()` will return an empty GlideRecord if there is no valid record. Use the `.isValidRecord()` method here instead.